command-snapshot.json

```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "alias": [],
+    "command": "data:create:file",
+    "flagAliases": [],
+    "flagChars": ["f", "i", "n", "o"],
+    "flags": ["api-version", "file", "flags-dir", "json", "name", "parent-id", "target-org"],
```

Comment:
@mshanemc - Is `name` meant to be an API or Developer Name, or more like a label?

Reply:
it's the name of the file once it gets uploaded to the org.  See the `examples`  where maybe the filename sucks and you want a better name to show up on the Files tab or the File related list.

You can see the effect of that in yesterday's demo https://drive.google.com/file/d/1U4UPL-bnRKU6D1paUmqVBoIIKLvWX4MP/view around 10:00

Reply:
In the UI, the name is actually labeled "Title":

<img width="790" alt="image" src="https://github.com/salesforcecli/plugin-data/assets/63259011/6b90b771-c357-4778-a88b-62f3022aeba0">

Reply:
@mshanemc - I hadn't noticed this comment from @jshackell-sfdc while I was writing my [other comment](https://github.com/salesforcecli/plugin-data/pull/922#discussion_r1603925110).

I think this is another good reason for changing the `--name` flag to `--title`.